src/screens/DataDisplayScreen.tsx

```diff
@@ -0,0 +1,45 @@
+
+import React, { useEffect, useState } from 'react';
+import { View, Text, FlatList, Button } from 'react-native';
+import { User, DataDisplayResponse } from '../types/Types';
+import { getDataDisplay } from '../apis/DataDisplayApi';
+
+const DataDisplayScreen: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const fetchData = async () => {
+    try {
+      console.log('Calling getDataDisplay'); // Log the getDataDisplay call
+      const response: DataDisplayResponse = await getDataDisplay();
+      console.log('Data Display Response:', response); // Log the data display response
+      setUsers(response.userDetailsList);
+    } catch (error) {
+      console.error('Error fetching data:', error);
+    }
+  };
+
+  return (
+    <View>
+      <Button title="Fetch Data" onPress={fetchData} /> // Add a button to trigger the fetchData function
```

Comment:
Remove the button "Fetch Data". Just fetch data and display whenever this screen is rendered